src/WindowSize.jsx

```diff
@@ -22,11 +21,15 @@ export class WindowSize extends React.Component {
   }
 }
 
+const mapStateToProps = state => ( {
+  printMode: state.view.printMode
```

Comment:
Is this used in the component?

Reply:
whoops, leftover from the loads of things i threw at this for testing